src/edit/DistortableCollection.Edit.js

```diff
@@ -209,6 +210,10 @@ L.DistortableCollection.Edit = L.Handler.extend({
       }
       if (data.status === 'complete') {
         clearInterval(updateInterval);
+
+        L.IconUtil.toggleXlink(link, 'get_app', 'spinner');
+        L.IconUtil.toggleTitle(link, 'Export Images', 'Loading...');
+        L.DomUtil.removeClass(link.firstChild, 'loader');
       }
       if (data.status === 'complete' && data.jpg !== null) {
```

Comment:
You could probably move the `data.jpg !== null` condition inside the `data.status === 'complete'` to make the code cleaner. 

Something like

```javascript
if (data.status === 'complete') {
        L.IconUtil.toggleXlink(link, 'get_app', 'spinner');
        L.IconUtil.toggleTitle(link, 'Export Images', 'Loading...');
        L.DomUtil.removeClass(link.firstChild, 'loader');

        if (data.jpg !== null) {
    
```